src/wymeditor/editor/image-handler.js

```diff
@@ -575,6 +579,7 @@ WYMeditor.ImageHandler.prototype._onImgMousedown = function (evt) {
     }
 
     // returning false here prevents drag of image
+    // except for in IE8
```

Comment:
I remember that `dragstart` is not triggered by dragging in `designMode` in IE. At least not in 8. Perhaps since 11, if at all.